packages/theme/modules/catalog/pages/category.vue

```diff
@@ -190,13 +190,13 @@ export default defineComponent({
 
     const { activeCategory } = useTraverseCategory();
     const activeCategoryName = computed(() => activeCategory.value?.name ?? '');
-    const routeData = ref<EntityUrl>({});
+    const routeData = ref<CategoryTree>(null);
```

Comment:
```suggestion
    const routeData = ref<CategoryTree | null>(null);
```

we don't have strict null checks in typescript but pedantism

Reply:
fixed.